ext/payflowpro/README.md

```diff
@@ -0,0 +1,44 @@
+# payflowpro
+
+![Screenshot](/images/screenshot.png)
+
+(*FIXME: In one or two paragraphs, describe what the extension does and why one would download it. *)
+
+The extension is licensed under [AGPL-3.0](LICENSE.txt).
+
+## Requirements
+
+* PHP v7.2+
+* CiviCRM (*FIXME: Version number*)
```

Comment:
Trivial but I don't like to see FIXME and think it would be better deleting various bits of the generated - they don't add anything when incomplete and for a core extension are not really going to be looked at much - they can be added/filled in later.

Reply:
This extension provides an integration with PayPal's PayFlow Pro service. It supports one time and recurring payments from CiviCRM.